app/views/devise/registrations/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class='container w-25 d-flex flex-column align-items-center p-3 mx-auto' >
 
```

Comment:
- [ ] OPTIONAL
Your app is functioning great, however, I think we can improve further on our design to look closer to the [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), so as make our app a good fit for showcasing on a portfolio. Kindly bear in mind the typography, color alignment, and spacing of objects.
- You can consider improving these pages:
  * Splash screen 
  * Sign up and login pages
  * Home page (categories page)
  * Transactions page
  * "Add a new category" page
  * "Add a new transaction" page